src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -92,6 +92,24 @@ DiagnosticBag diagnostics
             return new TupleTypeSymbol(underlyingType, elementNames, binder.Compilation.Assembly);
         }
 
+        internal static TupleTypeSymbol Create(
+            ImmutableArray<TypeSymbol> elementTypes,
+            ImmutableArray<string> elementNames,
+            CSharpCompilation compilation)
+        {
+            Debug.Assert(elementNames.IsDefault || elementTypes.Length == elementNames.Length);
+
+            int numElements = elementTypes.Length;
+
+            if (numElements <= 1)
+            {
+                throw ExceptionUtilities.Unreachable;
```

Comment:
This and our discussion today at LDM (treating `ValueTuple` and `(tuple)` the same) makes me think that the code will need to accept 0-tuples and 1-tuples, even if we don't offer language sugar for it.  I'll discuss with the team.